fcc-test-suite-template-master/css/style.css

```diff
@@ -0,0 +1,24 @@
+ .nav {
+     width: 100vw;
+ }
+
+ .nav .navbar {
+     padding-left: 25px;
+     padding-right: 25px;
+     width: 100%;
+ }
+
+ @media (max-width: 600px) {
+    .nav .navbar {
+        padding-left: 15px;
+        padding-right: 15px;
+        width: 80%;
+    }
+  }
+
+ 
+ .welcome {
+    height: 100vh;
+    background-color:lightgrey;
+    text-align: center;
+}
```

Comment:
Expected indentation of 2 spaces

Reply:
Kindly apply my first inline comment to reconstruct this too.